src/mimefactory.rs

```diff
@@ -275,7 +275,7 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
                         .unwrap_or_default()
                 }
             }
-            Loaded::MDN => ForcePlaintext::NoAutocryptHeader as i32,
+            Loaded::MDN => 0,
```

Comment:
i think this line also  forces encryption of the MDNs, not sure if this is desired currently.
i disabled that at some point as MDNs that cannot be decrypted are super-annoying for users - they do not know that it is "only" a MDN, ask back etc.

so, maybe better use ForcePlaintext::AddAutocryptHeader here?

Reply:
Fixed it, I did not intent to enable MDN encryption.